docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2020-10-12
+ - Added Content-Type application/health+json to list of expected
```

Comment:
```suggestion
 - Alert_on_Unexpected_Content_Types.js > Added Content-Type application/health+json to the list of expected types.
```

Reply:
Rebased, updated base and updated CHANGELOG.md